rfc/github-cdevents.md

```diff
@@ -0,0 +1,58 @@
+| dev.cdevents.branch.created      | create                      | A Git branch or tag is created.                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| dev.cdevents.branch.deleted      | push                        | A Git branch has been deleted. But to subscribe to only branch and tag deletions, use the delete webhook event.                                                                                                                                                                                                                                                                                                                                                   |
+| dev.cdevents.change.updated      | push                        | A commit/ commit tag is pushed.                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| dev.cdevents.change.*            | pull_request                | This event occurs when there is activity on a pull request. <details><summary>Actions</summary> {"Action type":{"assigned", "auto_merge_disabled", "auto_merge_enabled", "closed", "converted_to_draft", "demilestoned", "dequeued", "edited", "enqueued", <br/>"archived", "labeled", "locked", "milestoned", "opened", "ready_for_review","reopened", "review_request_removed", "review_requested", "synchronized", "unassigned", "unlabeled", "unlocked"}</br> |
+| dev.cdevents.change.*            | pull_request_review_comment | This event occurs when there is activity on a pull request review comment. <details><summary>Actions</summary> {"Action type":{"created", "reviewed", "commented"}                                                                                                                                                                                                                                                                                                |
```

Comment:
We need to map as much as possible events from GitHub to events that exist in CDEvents.